Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -30,6 +30,11 @@ For installing ``kubeadm`` and for more provisioning options please refer to
    maintained kernels, the situation could differ. Therefore, please check with
    your distribution.
 
+.. attention::
+
+   When switching Cilium with kube-proxy replacement from enabled to disabled,
+   or vice-versa, existing connections in that node will be disrupted.
```

Comment:
I think you want to specify which exactly connections will be disrupted, as it reads that all connections will be disrupted.

Reply:
Also, the quick start guide tells how to deploy a kube-proxy free version, and it will be using kpr=strict. I don't think we need to scare off users in here. This warning rather belongs into https://docs.cilium.io/en/v1.9/gettingstarted/kubeproxy-free/#kube-proxy-hybrid-modes where we talk about hybrid modes.

Reply:
>I think you want to specify which exactly connections will be disrupted,

@brb Which are?

>Also, the quick start guide tells how to deploy a kube-proxy free version, and it will be using kpr=strict. I don't think we need to scare off users in here.

@borkmann I'm not sure if I agree with that. The GSG recommends removing kube-proxy DS on running clusters:

> For existing installations with kube-proxy running as a DaemonSet, remove it by using the following commands:

This gives the impression that it is fine to remove `kube-proxy` without any consequences which is not correct.

Reply:
Yeah, that part of the guide looks actually a bit broken.. I just did a general edit. Please see: https://github.com/cilium/cilium/pull/16169